### PR TITLE
feat(service-registry artifact create): print url to UI

### LINKED ISSUE #1321: Creating a new artifact should return the URL to it
### Feature or problem description

When creating an artifact in Schema Registry in CLI, it should print the URL to UI to view the newly created artifact.

### Proposed solution

The artifact url in the ui looks something like this - `https://console.redhat.com/beta/application-services/service-registry/t/d5e6012f-ce90-4aaf-85b3-5f6e21a97e60/artifacts/default/rama-5/versions/latest`

URL format: `<registry-browser-url>/artifacts/<group>/<id>/versions/<version>`

### PR DESCRIPTION
**Pull request by rkpattnaik780**
Print URL to UI of the artifact created.

Closes #1321 

Screenshot from 2021-11-16 20-46-05

### Verification Steps

1. Create an artifact:
```
./rhoas service-registry artifact create  --file ./artifact.json --name rama-tes35  --type JSON --group schema --artifact-id rama-53909uj
```
It should display the web url for the artifact.

### Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation change
- [ ] Other (please specify)

### Checklist

- [X] Documentation added for the feature
- [X] CI and all relevant tests are passing
- [ ] Code Review completed
- [ ] Verified independently by reviewer

### CODE REVIEW

File: pkg/cmd/registry/artifact/crud/create/create.go [hunk 1]
```diff
@@ -111,6 +116,8 @@ func NewCreateCommand(f *factory.Factory) *cobra.Command {
 	cmd.Flags().StringVarP(&opts.artifactType, "type", "t", "", opts.localizer.MustLocalize("artifact.common.type", localize.NewEntry("AllowedTypes", util.GetAllowedArtifactTypeEnumValuesAsString())))
 	cmd.Flags().StringVar(&opts.registryID, "instance-id", "", opts.localizer.MustLocalize("artifact.common.instance.id"))
 
+	cmd.Flags().BoolVar(&opts.web, "web-url", false, opts.localizer.MustLocalize("artifact.common.webURL"))
```
[the remark this answers is not in the document]
Comment (rkpattnaik780) [reply pkg/cmd/registry/artifact/crud/create/create.go:119]: Requirements mentioned a flag, so got it mixed up. Will be rectifying that now.
I think we should display the API url as well.

[the remark this answers is not in the document]
Comment (wtrocki) [reply pkg/cmd/registry/artifact/crud/create/create.go:119]: I'm ok with flag but better UX will be to do it all the time. 
Let's leave flag but maybe improve message.

**Review by wtrocki — APPROVED** [review]

File: pkg/cmd/registry/artifact/crud/create/create.go [hunk 2]
```diff
@@ -177,5 +181,34 @@ func runCreate(opts *options) error {
...
+
+	registry, _, err := serviceregistry.GetServiceRegistryByID(opts.Context, conn.API().ServiceRegistryMgmt(), opts.registryID)
+	if err != nil {
+		return err
+	}
+
+	group := metadata.GetGroupId()
+
+	if group == "" {
+		group = "default"
```
Comment (craicoverflow) [inline pkg/cmd/registry/artifact/crud/create/create.go:206]: We should probably convert that to a constant.

File: pkg/cmd/registry/artifact/crud/create/create.go [hunk 3]
```diff
@@ -177,5 +181,34 @@ func runCreate(opts *options) error {
...
+	}
+
+	registryURL := fmt.Sprintf("%s/artifacts/%s/%s/versions/%s", *registry.BrowserUrl, group, metadata.Id, metadata.Version)
```
Comment (craicoverflow) [inline pkg/cmd/registry/artifact/crud/create/create.go:209]: Danger of a nil pointer exception occurring. Use the getter method for browser URL.

File: pkg/cmd/registry/artifact/crud/create/create.go (hunk 2 above)
Comment (wtrocki) [reply pkg/cmd/registry/artifact/crud/create/create.go:206]: There is one :D

Comment (craicoverflow) [reply pkg/cmd/registry/artifact/crud/create/create.go]: Yes but it is not being used here.

**Comment by wtrocki** [comment]
Good to merge!

File: pkg/cmd/registry/artifact/crud/create/create.go [hunk 4]
```diff
@@ -177,5 +181,34 @@ func runCreate(opts *options) error {
...
+	if err != nil {
+		return err
+	}
+
+	registry, _, err := serviceregistry.GetServiceRegistryByID(opts.Context, conn.API().ServiceRegistryMgmt(), opts.registryID)
+	if err != nil {
+		return err
+	}
+
+	group := metadata.GetGroupId()
```
Comment (craicoverflow) [inline pkg/cmd/registry/artifact/crud/create/create.go:203]: GetGroupIdOk is probably cleaner:

https://github.com/redhat-developer/app-services-sdk-go/blob/registryinstance/v0.3.1/registryinstance/apiv1internal/client/docs/ArtifactMetaData.md#getgroupidok

**Review by craicoverflow — APPROVED** [review]
Have not verified, but LGTM!

**Pull request merged**